acquia/cli
# DX-1750: pull:files from multisite apps

Motivation
----------
Running `acli pull:files` does not let you choose a specific site to pull from. It just pulls from the default site.

Proposed changes
---------
Let the user choose which site to pull files from.

Alternatives considered
---------
None, we have to support multisite.

Testing steps
---------
Ensure that `acli pull:files` works when selecting a multisite, and (still) works for ACSF and the default site.
